C2c cleanup (#3293)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md

1. Search for and select **Armis**.
1. Enter a **Name** for the Source. The description is optional.
1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
1. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a checkmark is shown when the field exists in the Fields table schema.
   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored (i.e., dropped).
1. In **Instance URL**, enter the Armis hostname.
   :::info
   Armis Instance URL is the Armis hostname. For example, `https://armis-instance.armis.com`.
   :::

> 1. In **Secret Key**, enter your API secret key that you have generated in [Setup and Configuration](#setup-and-configuration) section.

Reviewer: Broken link. There is no "Setup and Configuration" section.

### Configuration Object

| Parameter | Type | Required | Default | Description | Example |
|:---|:---|:---|:---|:---|:---|
| name | String | Yes | `null` | Type a desired name of the source. The name must be unique per Collector. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_source`. | `"mySource"` |
| description | String | No | `null` | Type a description of the source. | `"Testing source"`
| category | String | No | `null` | Type a category of the source. This value is assigned to the [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) field `_sourceCategory`. See [best practices](/docs/send-data/best-practices) for details. | `"mySource/test"`
| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
| instanceUrl | String | Yes | `null`| Armis Instance URL |  |
| apiSecretKey | String | Yes | `null`| Armis API secret key |  |

> | apiType | Array | Yes | | You may use either or both sources of data, i.e. Alerts and devices. |

Reviewer: ```suggestion
| apiType | Array | Yes | | You may use either or both sources of data, that is, Alerts and devices. |
```

    }
}
```

> ### Terraform example

Reviewer: There is no Terraform example.
